app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -579,10 +595,28 @@ class CommonTriageQuestionsController @Inject() (
                 propertyAddress = None,
                 disposalDetailsAnswers = None,
                 acquisitionDetailsAnswers = None,
-                reliefDetailsAnswers = None,
+                reliefDetailsAnswers =
+                  if (furtherReturn)
+                    Some(
+                      IncompleteReliefDetailsAnswers.empty.copy(
+                        otherReliefs = single.reliefDetailsAnswers.flatMap(_.fold(_.otherReliefs, _.otherReliefs))
```

Comment:
you should use `.unsetPrrAndLettingRelief` instead to make sure default values aren't overwritten for period of admin